auth.py

```diff
@@ -5,8 +5,13 @@
 def get_credentials():
     username = input('Enter your username: ')
     password = getpass.getpass('Enter your password: ')
-    return username, password
+    return username, hash_string(password)
 
+def hash_string(inputstring):
+    # hash the string
+    hashed_password = sum([ord(char) for char in inputstring]) 
```

Comment:
`[]` is not necessary, the sum could be just written as `sum(ord(char) for char in inputstring)`. This has the advantage that  a temporary list is not created.